Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore.py

```diff
@@ -11,52 +12,40 @@
     'URL': "url"
 }
 
-indicator = demisto.args()['indicator']
-resp = demisto.executeCommand("getIndicator", {'value': indicator})
 
-if True in [isError(entry) for entry in resp]:
-    demisto.results(resp)
-    sys.exit(0)
-
-data = demisto.get(resp[0], "Contents")
-
-if not data:
-    demisto.results("No results.")
-    sys.exit(0)
-
-ec = {}  # type: ignore
-ec["DBotScore"] = []
-
-for entry in data:
+def iterate_indicator_entry(indicator, entry):
     indicator_type = entry["indicator_type"]
-    score = entry["score"]
+    indicator_type = indicator_types.get(indicator_type, indicator_type)
     sources = entry.get('sourceBrands', [])
-    if hasattr(INDICATOR_TYPE_DICTIONARY, indicator_type):
-        indicator_type = INDICATOR_TYPE_DICTIONARY[indicator_type]
-
-    if sources:
-        for source in sources:
-            ec["DBotScore"].append({
-                "Indicator": indicator,
-                "Type": indicator_type,
-                "Vendor": source,
-                "Score": score
-            })
-    else:
-        ec["DBotScore"].append({
-            "Indicator": indicator,
-            "Type": indicator_type,
-            "Vendor": '',
-            "Score": score
-        })
-
-
-md = tableToMarkdown("Indicator DBot Score", ec["DBotScore"])
-
-demisto.results({
-    "Type": entryTypes["note"],
-    "ContentsFormat": formats["json"],
-    "Contents": ec,
-    "HumanReadable": md,
-    "EntryContext": ec
-})
+    sources = sources if sources else ['']
+    for source in sources:
+        dbot_score = Common.DBotScore(indicator=indicator, indicator_type=indicator_type,
+                                      integration_name=source, score=entry["score"]).to_context()
+        dbot_score = dbot_score.get(DbotScoreKey, dbot_score)
+        yield CommandResults(readable_output=tableToMarkdown('Indicator DBot Score', dbot_score)), dbot_score
```

Comment:
it will be easier to read if you reverse the order of the return values
since the first value is complex

It will also need to be updated in the calling function.
```suggestion
        yield dbot_score, CommandResults(readable_output=tableToMarkdown('Indicator DBot Score', dbot_score))
```